client/src/ui/NavBar.tsx

```diff
@@ -1,44 +1,154 @@
-import { Typography } from "@mui/material";
+"use client";
+import Image from "next/image";
 import Link from "next/link";
+import { Typography } from "@mui/material";
+import { usePathname } from "next/navigation";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faMessage,
+  faChartLine,
+  faMagnifyingGlass,
+} from "@fortawesome/free-solid-svg-icons";
+import { NavbarMenuDropdown } from "./NavbarMenuDropdown";
 
-interface ILinks {
```

Comment:
Why did you remove this? I've already set it up to make the NavBar easier to change?

Reply:
I was conforming to the already merged landingnavbar

Reply:
reinstated the array and use mapping to generate links for navigation